src/components/MenuPanel/OrganizationSwitcher/FavoriteRow.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { theme } from '@aragon/ui'
+import OrganizationItem from './OrganizationItem'
+import { FavoriteDaoType } from '../../../prop-types'
+
+class FavoriteRow extends React.Component {
+  static propTypes = {
+    dao: FavoriteDaoType.isRequired,
```

Comment:
Should this be of `DaoItemType` instead?

Reply:
Not this one: it needs the `favorited` flag :-)